### ISSUE TITLE
The remote directory isn't correctly added

### ISSUE THREAD

**Issue by gcetusic** [VorskiImagineering/C3PO#2]
Let's say we have a remote repository https://github.com/user/repo.git and it wasn't added to local git remote list. The following code should create the repository if 'git ls-remote' fails

```
try:
    subprocess.check_call(['git', 'ls-remote', git_repository])
except subprocess.CalledProcessError:
    try:
        subprocess.check_call(['git', 'remote', 'add', 'po_translator', git_repository])
    except subprocess.CalledProcessError as e:
        raise PODocsError(e)
```

Bit if the public key is set correctly, running 'git ls-remote git@github.com:user/repo.git' doesn't cause an exception and, later, the code fails because the repo po_translator doesn't exist.

**Comment by stefan8** [comment]
You are right. I suggest checking existing remote by remote show po_translator.

```
try:
    subprocess.check_call(['git', 'remote', 'show', '-n', 'po_translator'])
except subprocess.CalledProcessError:
    try:
        subprocess.check_call(['git', 'remote', 'add', 'po_translator', git_repository])
    except subprocess.CalledProcessError as e:
        raise PODocsError(e)
```

What do you think about this?
I can see problem when someone change git_repository in config, should we check if it's still the same? But probably it's not common case.

**Comment by gcetusic** [comment]
I ran into this problem using django-c3po so I'll give my opinion from that perspective. Since django-c3po has a setting GIT_REPOSITORY that's used when calling git_push, I think users expect THAT repository to be used. I'm guessing django-c3po users have no idea what exactly git_push does, shouldn't be bothered with git configuration beyond settings and C3PO should override .git/config since it added it in the first place so "remote show" line should be evicted entirely. Could you please review https://github.com/VorskiImagineering/C3PO/pull/3 ?
